spyder/plugins/findinfiles/widgets/results_browser.py

```diff
@@ -263,6 +263,18 @@ def clear_title(self, search_text):
         text = _('String not found')
         self.set_title(title + text)
 
+    def set_title(self, title):
+        metrics = QFontMetrics(self.font)
+        searchTextWidth = len(title)* metrics.width('W')
+        maxWidth = self.width()
+        if searchTextWidth >= maxWidth:
+            elided_title = metrics.elidedText(
+                title, Qt.ElideMiddle, maxWidth
+            )
```

Comment:
```suggestion
        search_text_width = len(title) * metrics.width('W')
        max_width = self.width()
        if search_text_width >= max_width:
            elided_title = metrics.elidedText(title, Qt.ElideMiddle, max_width)
```

* We don't tend to use camel case variables but snake case ones instead.
* Fix formatting because the `elide_title = ...` fits into 79 columns.